src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -23,36 +23,44 @@
 Yast.import "UI"
 
 module UpdateAlternatives
-
+  # Dialog where all alternatives groups in the system are listed.
   class MainDialog < UI::Dialog
-
     def dialog_options
       Opt(:decorated, :defaultsize)
     end
 
     def dialog_content
       VBox(
         create_table,
-        HBox(
-          PushButton(Id(:show), _("Show alternatives")),
-          PushButton(Id(:cancel), _("Cancel"))
-        )
+        footer
       )
     end
 
     def show_handler
       AlternativesDialog.new.run
     end
 
+    def alternatives_table_handler
+      AlternativesDialog.new.run
```

Comment:
so change of item in table immediatelly open dialog? I am not sure if it is good idea and also a bit inconsistent with other modules ,where you select item and then action ( clicking on button ).